### PR TITLE
Hid unused menu items, added german for searchbox

### PR DESCRIPTION
**Pull request by mayank-cronj**
implements OpusCapita/bnp#103

### CODE REVIEW

File: src/SidebarMenu/SidebarMenu.js [hunk 1]
```diff
@@ -102,7 +101,7 @@ export default class SidebarMenu extends React.Component {
           <div className="nav-background" />
           <div className="navbar-header hidden-md">
             <a className="navbar-brand visible-lg" href="http://www.opuscapita.com/">
-              <img src={logo} style={{ height: '1.4em' }} />
+              <img src="" style={{ height: '1.4em' }} />
```
Comment (patrykkopycinski) [inline src/SidebarMenu/SidebarMenu.js]: hmm? how it's going to work without `src`?

**Review by patrykkopycinski — CHANGES_REQUESTED** [review]

File: src/SidebarMenu/SidebarMenu.js (hunk 1 above)
Comment (mayank-cronj) [reply src/SidebarMenu/SidebarMenu.js]: My bad. Had removed while testing. Fixed in new commit

File: src/HeaderMenu/i18n/de.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export default {
   HeaderMenu: {
+  	search: 'Suche',
```
Comment (patrykkopycinski) [inline src/HeaderMenu/i18n/de.js]: wrong indentation

**Review by patrykkopycinski — CHANGES_REQUESTED** [review]

File: src/HeaderMenu/i18n/de.js (hunk 2 above)
Comment (mayank-cronj) [reply src/HeaderMenu/i18n/de.js]: Followed the same indentation from the other translation files. Could you tell me what needs to be changed?

**Review by gr4per — APPROVED** [review]
lgtm
we never set strict formatting rules, but
1. no tabs
2. 2 spaces per indent
3. don't change formatting in code files when not doing semantical changes, this is to keep diffs small

That being said, this is ok for now, today no time to argue about indents

**Pull request merged**